tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -883,30 +889,34 @@ def test_positive_provision_rhev_with_image(self):
                     ['Host', 'Compute profile', COMPUTE_PROFILE_LARGE],
                     ['Host', 'Puppet Environment',
                         self.config_env['environment']],
-                    ['Operating System', 'Operating System', self.rhev_img_os],
+                    ['Operating System', 'Operating System', self.os_name],
                     ['Operating System', 'Partition table',
                         self.config_env['ptable']],
                     ['Operating System', 'PXE loader', 'PXELinux BIOS'],
                     ['Operating System', 'Root password', root_pwd],
                 ],
                 provisioning_method='image',
             )
-            self.assertIsNotNone(self.hosts.search(
-                u'{0}.{1}'.format(hostname.lower(),
-                                  self.config_env['domain']
-                                  )))
+            vm_host_name = '{0}.{1}'.format(
+                hostname.lower(), self.config_env['domain'])
+            # the provisioning take some time to finish, when done will be
+            # redirected to the created host
+            # wait until redirected to host page
+            self.assertIsNotNone(
+                session.hosts.wait_until_element(
+                    locators["host.host_page_title"] % vm_host_name,
+                    timeout=160
```

Comment:
Increased Timeout because the cloning of the template requires more time depending on our INFRA res